webapp/packages/core-theming/src/styles/_form-controls.scss

```diff
@@ -33,9 +33,11 @@
       width: 16px;
     }
 
-    &:global([disabled]),
-    &:global([readonly]) {
-      opacity: 0.8;
+    &[type="text"] {
+      &:global([disabled]),
+      &:global([readonly]) {
+        opacity: 0.8;
+      }
```

Comment:
probably affects too many places, better to revert and make a fix for a specific place
```
some-selector {
  opacity: 1 !important;
}
```